apps/oidc-client-test/src/main.ts

```diff
@@ -54,7 +54,7 @@ async function bootstrap() {
   await app.listen(3001);
 }
 
-OpenIdClient.build(OIDC_CLIENT_METADATA_JWT, OIDC_CLIENT_PARAMS, OIDC_IDP_ISSUER_URL)
+OpenIdClient.build(OIDC_CLIENT_METADATA_BASIC, OIDC_CLIENT_PARAMS, OIDC_IDP_ISSUER_URL)
```

Comment:
What the difference between jwt and basic metadata?

Reply:
It's just how the client requests an authentication method. It was always supposed to use OIDC_CLIENT_METADATA_BASIC.

Reply:
It just determines what authentication method the client wants to use. It should have stayed as OIDC_CLIENT_METADATA_BASIC.